README.md

```diff
@@ -78,34 +47,44 @@ To avoid discussion the use of the above HTTP methods is described below.
 #### `POST`
 - Used to create a resource.
 - Does not return the same response on repeated requests.
-- Does not need the full resource for the request (often does not need the id for the to be created resource.)
+- Does not need the full resource for the request (often does not need the id for the to be created resource).
 
 #### `PATCH`
 - Used to update an existing resource.
 - Does not return the same response on repeated requests.
 - Does not need the full resource for the request (you might only want to update a user's name for example).
 
 ### API versioning
-The API versioning follows semantic versioning. The increment in version number is **not** done manually and should not be part of the pull request.
+The API versioning follows semantic versioning. The increment in version number is done manually and should be part of the pull request.
 
-### Definition file naming
+### Schema file naming
 Definition file names follow PascalCasing. Where every first letter of a word (including the first word) is uppercase. For example, the definition for a country code would be found in:
+
 ```
-specification/definitions/CountryCode.json
+specification/schemas/CountryCode.json
 ```
 
 ### Path file naming
-The files in `specification/paths` are named to their corresponding API endpoints. Where resources start with an uppercase letter and path variables with a lowercase letter. For example, the definition of the following route: 
+The files in `specification/paths` are named to their corresponding API endpoints. Where resources start with an uppercase letter and path variables with a lowercase letter. For example, the definition of the following route:
```

Comment:
I would say: "The files in `specification/paths` are named **after** their corresponding API endpoints.